node/Cargo.toml

```diff
@@ -13,6 +13,7 @@ attestation = { workspace = true }
 axum = { workspace = true }
 backon = { workspace = true }
 borsh = { workspace = true }
+bincode = { workspace = true }
```

Comment:
Oh this PR introduces bincode serialization? As much as I love bincode, it's good if we're consistent in which format we use to serialize the data in our database. Is there any reason not to go with JSON serialization?

Secondly, if we want a separate binary format here I'd also consider borsh over bincode because we already use it for our smart contracts and it has a strict specification.

Reply:
> Not sure why I used bincode, probably skill issue.

Wouldn't call it a skill issue. In isolation it's a good choice. Bincode is probably (most certainly) a better choice than serde_json overall. It's just a matter of consistency.